data/human/human missions.txt

```diff
@@ -4077,3 +4077,135 @@ mission "Laborer Payment"
 		conversation
 			`You receive a payment of <payment> from the down-on-his-luck laborer that you transported a while back, along with a message. "Hello <last>. Not sure if you remember me. I'm the one that had to deal with my boss bad-talking me to everyone on my home planet. I've been doing better, but my new boss keeps trying to rip me off and make me come in for overtime." He goes into detail about the sordid working conditions in his new job, but you stop reading after the third paragraph.`
 				decline
+
+
+mission "Riley"
+	name "Visiting Home"
+	description "Take a woman named Riley to see her ailing father on <destination> before it's too late."
+	minor
+	passengers 1
+	source
+		attributes deep 
+		attributes urban
+	deadline
+	destination "Memory"
+	to offer
+		random < 15
+	on offer
+		conversation
+			`Entering the spaceport, you notice a young woman moving from captain to captain, probably looking to hire one.`
+			choice
+				`	(Ask the young woman if you can help her.)`
+				`	(Ignore her.)`
+					decline
+			`	You approach the woman and ask her if she is looking to hire a pilot.`
+			`	"I need to get to <destination> to see my father. He's dying. I have <payment> credits to pay for the journey; please, will you help me? I need to be there in <deadline> days, at the most."`
```

Comment:
```suggestion
			`	"I need to get to <destination> to see my father. He's dying. I have 13,000 credits to pay for the journey; please, will you help me? I need to be there by <day>, at the most."`
```
Looks like <payment> in an `on offer` conversation will look for the last payment action, even if it's optional, compared to the guaranteed payment. Seems like a bug, but for now, it'll need to be hardcoded here.

No <deadline> tag either, you either get a full date with <date>, or a shortened version of that with <day>.